### PR TITLE
Added fields to the Resource model and added image utilities

### PR DESCRIPTION
**Pull request by brittanyberlanga**
Originally the plan was to just add author, description, and image fields to the resource model; however, I noticed that many of the model objects with image fields declare many of the same functions and I thought making a utilities module for all of them would be useful.

I ran the server locally to make sure my changes didn't break anything, but since I can't actually upload a photo to aws, I wasn't able to fully test everything.

Let me know if you see anything concerning :)

### CODE REVIEW

**Comment by fedorareis** [comment]
not to be nitpicky, but could you fix the styling (mainly the indentation) in the resources file. It is pretty all over the place.

**Comment by brittanyberlanga** [comment]
@fedorareis Yeah, of course! What specifically needs fixing? Is it the 3 space indentations compared to 2 space indentations?

**Review by fedorareis — CHANGES_REQUESTED** [review]
There are a number of spots where the closing brace is indented one too few characters. There are a couple instances of lines not being indented to the level they should be (see line 93 of Resources.js).

**Comment by tylerbwong** [comment]
@fedorareis Resources.js doesn't appear to have a 93rd line. Is there a different example?

File: models/Resource.js [hunk 1]
```diff
@@ -13,12 +14,114 @@ var Resource = new keystone.List('Resource', {
...
+	squareImageLink: {
+		type: Types.Url,
+		hidden: false,
+		noedit: true,
+		watch: true,
+		value: function() {
+	      return this.squareImage.url ? 'https:' + this.squareImage.url : '';
+	   },
+		format: function(url) {
+      	return url;
```
Comment (fedorareis) [inline models/Resource.js]: This was the line I was trying to reference. THe return should be indented 2 more times and the curly brace below should be indented one more time.

**Review by fedorareis — CHANGES_REQUESTED** [review]
Most of the spots where there is a return in this file the indentation is wrong. Honestly I would recommend just running it through an formatter or an editor with formatting rules set. We are using the standard Node coding style if I remember correctly.

File: models/Resource.js (hunk 1 above)
Comment (tylerbwong) [reply models/Resource.js]: @fedorareis I'm pretty sure you're looking at an old commit. The most recent one should be from 2 days ago.

Comment (tylerbwong) [reply models/Resource.js]: 3d248944d2dc3cc417d5073c786580b7c1c0d9cb is the most recent commit hash.

Comment (fedorareis) [reply models/Resource.js]: whoops, totally didn't see that. Then from what I can see everything looks good.

**Review by fedorareis — APPROVED** [review]

**Comment by fedorareis** [comment]
I will push the updates live this weekend.

**Pull request merged**